### DISCUSSION TITLE
Issues reloading packages in R

### DISCUSSION THREAD

**Discussion by 2199105T** [IBAHCM/RPiR#117] [Archive] [answered]
Hi @richardreeve,

I had put in an error code that was coming up in my R Studio when trying to reinstall the R packages in the zoom chat during the break. I was wondering if someone could come to my breakout room and I can share my screen to get help like you suggested? I am in Room 5.

Thanks,

Sarah

**Comment by richardreeve** [comment, marked as answer]
Hopefully that's now fixed. As a general point for people with similar problems, it is almost always the right answer to reinstall any package that is causing problems using `install.packages("troublesome_package")`.

**Reply by richardreeve** [reply]
If that turns out not to be enough, see more tips on installation issues at https://ibahcm.github.io/RPiR/articles/pages/install_RPiR.html#potential-problems
